examples/seasonal-journey/service.bal

```diff
@@ -0,0 +1,80 @@
+// AUTO-GENERATED FILE. DO NOT MODIFY.
```

Comment:
I assume that this file is not autogenerated as this is an example. If so do we need this header ?